engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -117,17 +142,26 @@ public Vector3f getExtents() {
         return dimensions;
     }
 
+    /**
```

Comment:
Description!

Reply:
Even the most trivial methods must have a description

Reply:
Same for all other get functions

Reply:
I would like to see javadoc where it adds more information which is not visible in the API, for example what elements are expected to be in a passed list, if a method can handle null, if a method has side-effects, all these kind of things.
Javadoc which repeats the method name like
```
/**
* Getter for foo.
* @return the foo.
**/
public Foo getFoo();
```
are just 4 lines of wasted space and 10 seconds wasted of developer time, they add zero new information.

Reply:
@oniatus Excellent point. Should I remove the docs in all such cases?